<launch>
  <node name="S300" pkg="pacecat_s300" type="pacecat_s300_node" output="screen">

    <param name="frame_id" value="map" />

    <param name="topic_pointcloud" value="pointcloud" />
    <param name="output_pointcloud" value= "true"/>

    <param name="topic_custommsg" value="custommsg" />
    <param name="output_custommsg" value="false"/>

    <param name="topic_imu" value="imu" />
    <param name="output_imu" value="true"/>
   
    <param name="lidar_port" value="6001" />
    <param name="local_port" value="6002" />
    <param name="lidar_ip" value="192.168.137.200" />
  </node>
</launch>
